full_install.sh

```diff
@@ -4,12 +4,25 @@ set -euo pipefail
 # Get the directory of the script
 script_dir=$(dirname "$0")
 
-# Change the current directory to the script's directory
-cd "$script_dir"
+# Check if we're in Colab
+IN_COLAB=$(python -c "
+try:
+    import google.colab
+    print('true')
+except ImportError:
+    print('false')
+")
 
-EDITABLE="true"
-UV_FLAGS=""
+# Set installation type based on environment
+if [ "$IN_COLAB" == "true" ]; then
+    EDITABLE="false"
+    echo "Colab detected - forcing non-editable install"
+else
+    EDITABLE="true"
+    cd "$script_dir"
```

Comment:
can we try moving the `cd "$script_dir"` outside of the if/else, so it always happens? This should fix the below:

Ideally this should work in colab:

```
!git clone -b revert_change https://github.com/tonyhoo/autogluon.git
!./autogluon/full_install.sh
```

But it doesn't currently:

```
error: Distribution not found at: file:///content/common
```